src/app/enums/pile.enum.js

```diff
@@ -1,4 +1,6 @@
 export const PileEnum = {
-    DrawPile: 'Draw Pile',
-    DiscardPile: 'Discard Pile',
+    DrawPile: 'drawPile',
+    DiscardPile: 'discardPile',
+    HumanHand: 'humanHandPile',
+    BotHand: 'botHandPile'
```

Comment:
Why did you change these to lowercase?

Reply:
i also added the two hand piles... 
i did it because i needed them to be equal to the fields of state in the "board.component.js" file

Reply:
in lines 12-15 on "board.component.js"

Reply:
ok